src/pages/haeriRoh/Login/LoginHr.scss

```
@@ -0,0 +1,135 @@
+$fa-color: #fafafa;
+$wh-color: white;
+$focus-line: 1px solid gray;
+$border-style: 1px solid #dbdbdb;
+
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* css code */
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  width: 100%;
+  height: 100vh;
+  @include flex-center();
+  background-color: $fa-color;
+
+  .container {
+    width: 340px;
+    @include flex-center();
+  }
+
+  .lineBox {
+    width: 320px;
+    height: 370px;
+    border: $border-style;
+    background-color: $wh-color;
+  }
+
+  .contentsBox {
+    width: 100%;
+    height: 100%;
+
+    .logo {
+      width: 100%;
+      height: 130px;
+      @include flex-center();
```

```suggestion
      @include flex-center;
      width: 100%;
      height: 130px;
```
mixin을 활용할 때는 스타일 속성에 가장 윗부분에 사용해 주는 것이 좋습니다.